docs/remote-deployment.rst

```diff
@@ -0,0 +1,170 @@
+-----------------------------
+
+Once the config properties file is updated with the remote TorchServe instance details. Set the environment variable CONFIG_PROPERTIES  with the path of the config.properties file.
+
+For example:
+
+.. code-block::
+
+    export CONFIG_PROPERTIES=/home/ubuntu/mlflow-torchserve/config.properties
+
+Install mlflow-torchserve Plugin:
+---------------------------------
+
+Ignore this step if the mlflow-torchserve plugin is already installed.
+
+Install torchserve plugin using the following command
+
+.. code-block::
+
+    pip install mlflow-torchserve
+
+Creating a new deployment:
+--------------------------
+
+Creating a deployment in the remote TorchServe is a two step process.
+
+    1. Generate mar file using create deployment script
+    2. Register the model using the mar file.
+
+Run the following script to start with the deployment process.
+
+.. code-block::
+
+    python create_deployment.py --deployment_name mnist_test
+
+This comment will generate a `mnist_test.mar` file inside the `model_store` folder.
+
+Since, the model needs to be deployed on the remote torchserve, the mar file needs to be exposed as a public url. 
+
+Here is an example of hosting the mar file using python http server and ngrok. Any alternate mechanism can be used to expose the mar file as public url (For ex: uploading it into a S3 bucket and assigning necessary permissions to download it from http/https url).
+
+Start the http server from model store as below
+
+.. code-block::
+
+    cd model_store
+    python -m http.server
+
+This is to host the file in the local instance. 
+The verification can be done by downloading the file from the browser or from terminal using wget.
+
+Open the browser and hit - `http://localhost:8000/mnist_test.mar <http://localhost:8000/mnist_test.mar>`_
+
+Or in the terminal do `wget` `http://localhost:8000/mnist_test.mar <http://localhost:8000/mnist_test.mar>`_
+
+The mnist.mar file will be downloaded. However, remote torchserve instance, doesnt understand the mar file hosted in localhost. 
+
+Download and unzip ngrok file from the following url - `https://ngrok.com/download <https://ngrok.com/download>`_
+
+Run the following command to run ngrok -
+
+.. code-block::
+
+    ./ngrok http 8000
+
+Copy the web address from the forwarding section and update the EXPORT_URL parameter in config.properties file.
+
+For example:
+
+.. code-block::
+
+    inference_address=http://18.225.35.109:8080
+    management_address=http://18.225.35.109:8081
+    export_url= http://eda154810618.ngrok.io
+
+This is an optional step added for verification.
+
+Download the mar file using ngrok url . Open browser and hit
+
+.. code-block::
+
+    http://eda154810618.ngrok.io/mnist_test.mar
+
+mnist_test.mar file should be downloaded.
+
+We are all set for performing registration. To register the model in remote torchserve instance run 
+
+.. code-block::
+
+    python register.py --mar_file_name mnist_test.mar
```

Comment:
@shrinath-suresh Where is the register.py file located? Not seeing it in the MNIST example folder

Reply:
@chauhang We no longer need a separate script for registering the model. As discussed, the model registration can be done by adding a parameter in log_model . I have updated the code to perform registration in `mnist_model.py` itself - https://github.com/mlflow/mlflow-torchserve/blob/70baee835723b0b61421ed37840a999bf7eb1ec6/examples/MNIST/mnist_model.py#L301

I will update the documentation.

Reply:
I have updated the steps

Reply:
I got confused with model registration and torchserve mar file registration and deleted the file. I have added it back - https://github.com/mlflow/mlflow-torchserve/pull/47/files#diff-a408565ad545aced7e32600c579119784fa4b1d88867d62205f44e1806a052a4